packages/playwright/src/tests/Profilediting.spec.ts

```diff
@@ -0,0 +1,13 @@
+import { test, expect } from '@playwright/test';
+
+test('test', async ({ page }) => {
+  await page.goto('https://kunsthalte.vercel.app/nl');
+  await page.getByRole('link', { name: 'Inloggen' }).click();
+  await page.getByTestId('input-email').click();
```

Comment:
It's not that important but in general the correct action for input elements is `focus`.

```suggestion
  await page.getByTestId('input-email').focus();
```

Reply:
I noticed that I had used the same `click` event in Login.ts page  🙈